.github/workflows/package-publish.yml

```diff
@@ -0,0 +1,125 @@
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.x"
+    - name: Install poetry build
+      run: poetry install
+
+    - name: Build a binary wheel and a source tarball
+      run: poetry build
```

Comment:
what about minimal qat compiler build ?

Reply:
doing that after this, just want the core mechanism in place first